Dockerfile

```diff
@@ -211,6 +214,7 @@ COPY --from=git-dev --chown=www-data:www-data /opt/kimai /opt/kimai
 RUN export COMPOSER_HOME=/composer && \
     composer install --working-dir=/opt/kimai --optimize-autoloader && \
     composer clearcache && \
+    composer require --working-dir=/opt/kimai laminas/laminas-ldap && \
```

Comment:
idk how composer works inside, but i think cache clearing should be the last step?

Reply:
doesn't matter.

Reply:
ok thy for this info